ppr-ui/src/components/parties/composables/useDebtorValidation.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  ref
+} from '@vue/composition-api'
+import { createDefaultValidationResult } from '@lemoncode/fonk'
+// import { formValidation } from './collateralFormValidator'
+
+const createEmptyErrors = () => ({
+  businessName: createDefaultValidationResult(),
+  first: createDefaultValidationResult(),
+  last: createDefaultValidationResult(),
+  street: createDefaultValidationResult(),
+  city: createDefaultValidationResult(),
+  region: createDefaultValidationResult(),
+  postalCode: createDefaultValidationResult()
+})
+
+export const useDebtorValidation = () => {
+  const errors = ref(createEmptyErrors())
+
+  /* const validateInput = (fieldName, value) => {
```

Comment:
can all this commented out stuff be deleted or is it going to be used later or something?

Reply:
going to be used later when validations are present :)